cli/i18n/I18nFile.php

```diff
@@ -115,10 +117,10 @@ private function unflatten(array $translation): array {
 		$a = array();
 
 		ksort($translation, SORT_NATURAL);
-		foreach ($translation as $compoundKey => $value) {
+        foreach ($translation as $compoundKey => $value) {
```

Comment:
Please revert those two changes (just above and below), unrelated to this PR.
And the first is a regression anyway